test-data/client-cache/cache_aa.rpl

```diff
@@ -0,0 +1,198 @@
+; Test if caching clears the AA flag. We issue the same query twice.
```

Comment:
If these rpl files are taken as-is from Unbound it might be good to a comment referencing the origin.
If these files have been modified compared to the original that would also be useful to note.

Reply:
The tests are new. Some of the boiler plate for a recursor has been stolen. I'm not sure if it makes sense to remove the boiler plate or not